packages/node_modules/@ciscospark/plugin-phone/src/phone.js

```diff
@@ -244,17 +244,20 @@ const Phone = SparkPlugin.extend({
    * @returns {undefined}
    */
   onLocusEvent(event) {
-    // We only want to handle calls we are not aware of
-    if (this.emittedCalls.has(event.data.locus)) {
+    const {locus} = event.data;
+
+    // We only want to handle calls we are not aware of in an active state
+    if (this.emittedCalls.has(locus) || locus.fullState.state.toLowerCase() === 'inactive') {
```

Comment:
Based on the flows we covered a couple weeks ago, it is possible to go from `inactive` to something else and that behavior should be reflected by events. If that's the case, will this logic still work for that condition?

Reply:
The actual call object will continue to emit events as locus events, but the phone plugin will not emit new call events (`call:created` and `call:incoming`) for inactive locus objects.

Reply:
Ahhh, that's right. Call locus events are handled elsewhere.